config/qiskit-aer-gpu-0.9.1.sh

```diff
@@ -0,0 +1,22 @@
+PACKAGE="qiskit-aer"
+PACKAGE_SUFFIX="-gpu"
+# setup.py takes package name from QISKIT_AER_PACKAGE_NAME environment
+# variable, so:
+PRE_BUILD_COMMANDS="export QISKIT_AER_PACKAGE_NAME=qiskit-aer-gpu"
+PACKAGE_DOWNLOAD_ARGUMENT="$PACKAGE==$VERSION"
```

Comment:
This is already the default, no need to specify it.

Reply:
The default is indeed "$PACKAGE==$VERSION", but I redefine the PACKAGE variable, so I have to redefine it. If I delete this lilne, it tries to download qiskit-aer-gpu==0.9.1 which does not provide source distribution. Source distribution is in qiskit-aer==0.9.1.

Reply:
Ok